scripts/install.sh

```diff
@@ -196,14 +196,23 @@ info "Setting kube context to use namespace: $namespace"
 kubectl config set-context --current --namespace="$namespace"
 
 info "Getting ModelMesh Serving configs"
-if [[ -n $install_local_path ]]; then
-  if [[ -f $install_local_path ]] && [[ $install_local_path =~ \.t?gz$ ]]; then
-    tar -xf "$install_local_path"
-    cd "$(basename "$(basename $install_local_path .tgz)" .tar.gz)"
-  elif [[ -d $install_local_path ]]; then
-    cd "$install_local_path"
+if [[ -n $install_config_path ]]; then
+  if [[ -f $install_config_path ]] && [[ $install_config_path =~ \.t?gz$ ]]; then
+    tar -xf "$install_config_path"
+    cd "$(basename "$(basename $install_config_path .tgz)" .tar.gz)"
+  elif [[ $install_config_path =~ ^http.+ ]]; then
+    if [[ $install_config_path =~ \.t?gz$ ]]; then
+      curl -L $install_config_path -O
+      filename=$(basename $install_config_path)
+      tar -xf "$filename"
+      cd "$(basename "$(basename $filename .tgz)" .tar.gz)"
+    else
+      die "Provided URL should be a .tgz or tar.gz archive file"
+    fi
+  elif [[ -d $install_config_path ]]; then
+    cd "$install_config_path"
```

Comment:
When I provide the "ModelMesh Serving directory", it fails later with "cd: default: No such file or directory".

There are two possible solutions:
1. change the description to "ModelMesh Serving config directory"
2. add "cd config" wherever applicable